runtimes/standalone/src/lib.rs

```diff
@@ -30,7 +30,7 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 #[cfg(feature = "runtime-benchmarks")]
 use frame_system::EnsureSigned;
 
-use did::DidSignature;
+use did::{DidRawOrigin, DidSignature};
```

Comment:
M: I guess this was automatically imported. There is now a mixed usage of `DidRawOrigin` and `did::DidRawOrigin`.

Reply:
2f80b2d